lib/mix/test/mix/tasks/xref_test.exs

```diff
@@ -1127,6 +1209,68 @@ defmodule Mix.Tasks.XrefTest do
       """)
     end
 
+    test "group with sink (json)" do
```

Comment:
IMO we don't need this test. The group works irrespective of the format.

Reply:
ack.